integration/service/oasis/stationfinder/cloudfinder/orig_station_finder_test.go

```diff
@@ -10,7 +10,9 @@ import (
 func TestOrigStationFinderIterator(t *testing.T) {
 	sf := CreateMockOrigStationFinder1()
 	c := sf.IterateNearbyStations()
-	var r []stationfindertype.ChargeStationInfo
+	var r []*stationfindertype.ChargeStationInfo
+
+	isdoneC := make(chan bool)
 	go func() {
```

Comment:
Why not remove `go func()`, go with the `for` loop directly?

Reply:
Will go with that.